### DISCUSSION TITLE
TLS certs expiring again?

### DISCUSSION THREAD

**Discussion by kkashtan-intel** [community/community#53435] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Seems very much like in previous thread month ago [https://github.com/orgs/community/discussions/50966](https://github.com/orgs/community/discussions/50966)

 Server certificate:
        subject: C=US; ST=California; L=San Francisco; O=GitHub, Inc.; CN=github.com
        **start date: 2022-04-20 00:00:00 GMT**
        **expire date: 2023-04-20 23:59:59 GMT**
        issuer: C=US; O=DigiCert Inc; CN=DigiCert TLS RSA SHA256 2020 CA1
       SSL certificate verify result: certificate has expired (10), continuing anyway.

**Comment by airtower-luna** [comment]
The certificate you see is very different from the one I see, even the issuer is different:

```
 0 s:C = US, ST = California, L = San Francisco, O = "GitHub, Inc.", CN = github.com
   i:C = US, O = DigiCert Inc, CN = DigiCert TLS Hybrid ECC SHA384 2020 CA1
   a:PKEY: id-ecPublicKey, 256 (bit); sigalg: ecdsa-with-SHA384
   v:NotBefore: Feb 14 00:00:00 2023 GMT; NotAfter: Mar 14 23:59:59 2024 GMT
```

The SHA-256 fingerprint of the certificate I see is: 92:A3:7F:BD:5E:21:A5:3A:95:C7:16:E1:14:4F:44:2F:58:2B:94:D0:FA:FC:67:3E:B6:71:7A:4E:B5:1A:88:A7

I can't tell you what's going on, but it's obvious they didn't just forget to renew.

**Comment by kkashtan-intel** [comment]
This is weird, seems fixed? Started working again, but the issuer is still SHA256 one
Server certificate:
        subject: C=US; ST=California; L=San Francisco; O=GitHub, Inc.; CN=github.com
        start date: 2023-04-21 00:00:00 GMT
        expire date: 2024-04-23 23:59:59 GMT
        issuer: C=US; O=DigiCert Inc; CN=DigiCert Global G2 TLS RSA SHA256 2020 CA1
        SSL certificate verify ok.

**Comment by kkashtan-intel** [comment]
Seems it's flickering right now - the old and the new cert is displayed intermittently when doing curl:

curl -vvvv -k https://github.com/ 2>&1|grep "start date"
       start date: 2023-04-21 00:00:00 GMT
curl -vvvv -k https://github.com/ 2>&1|grep "start date"
       start date: 2022-04-20 00:00:00 GMT
